packages/fluent-theme/src/components/activity/ActivityDecorator.module.css

```diff
@@ -88,15 +88,15 @@
     margin-inline-start: 28px;
   }
 
-  :global(.border-loader) {
+  :global(.border-loader):global(.border-loader) {
```

Comment:
Dupe?

Reply:
We need to increase specificity

Reply:
I thought about you intentionally do it too.

Reply:
I mean it should work for variant classes, but I choose to do it explicitly